test/jdk/java/awt/Multiscreen/MultiScreenInsetsTest/MultiScreenInsetsTest.java

```diff
@@ -79,14 +82,42 @@ public static void main(String[] args) throws InterruptedException {
 
             Rectangle frameBounds = frame.getBounds();
             frame.dispose();
+
+            /*
+             * On Windows, the top-left corner of an undecorated maximized frame may have negative coordinates (x, y).
+             * Adjust the frame bounds accordingly.
```

Comment:
This line is too long, we usually limit the number of columns to 80.
```suggestion
             * On Windows, the top-left corner of an undecorated maximized frame
             * may have negative coordinates (x, y).
             * Adjust the frame bounds accordingly.
```
Alternatively, you may wrap before _“frame”_.